azure-pipelines.yml

```diff
@@ -211,6 +211,9 @@ stages:
           displayName: Restore
           condition: succeeded()
 
+        - powershell: eng\SetupVSHive.ps1
```

Comment:
❓Is there a particular reason this moved?

Reply:
This is more of a "setup the machine" action so I grouped it with similar actions.